sdk/appconfiguration/azure-appconfiguration-provider/samples/refresh_decorator_sample.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+from azure.appconfiguration.provider import load, AzureAppConfigurationRefreshOptions
+from azure.appconfiguration import (
+    AzureAppConfigurationClient,
+    ConfigurationSetting,
+)
+import os
+import time
+
+connection_string = os.environ.get("AZURE_APPCONFIG_CONNECTION_STRING")
+
+# Setting up a configuration setting with a known value
+client = AzureAppConfigurationClient.from_connection_string(connection_string)
+
+configuration_setting = ConfigurationSetting(key="message", value="Hello World!")
+
+client.set_configuration_setting(configuration_setting=configuration_setting)
+
+# Connecting to Azure App Configuration using connection string
+refresh_options = AzureAppConfigurationRefreshOptions()
+refresh_options.refresh_interval = 1
+refresh_options.register(key_filter="message")
+config = load(connection_string=connection_string, refresh_options=refresh_options)
+
+
+@config.drefresh  # cspell:disable-line
```

Comment:
Typo in decorator?

Reply:
Placeholder name to we have a better one...

Reply:
Updated to `refresh_configuration_settings` for now.